app/views/admin/loans/timeline/_timeline.html.slim

```diff
@@ -0,0 +1,8 @@
+section.timeline
+  div.project-steps
+    - unless @loan.project_steps.empty?
+      - @loan.project_steps.each do |step|
+        = render "admin/loans/timeline/project_step", step: step
+    - else
+      // TODO: This will show a new form for project step
```

Comment:
In a separate feature issue, this will display a form for a new project step. This TODO comment can be removed.